### PR TITLE
LPS-116387 Add workflow support for account roles

### PR DESCRIPTION
**Pull request by pei-jung**
Hey Drew,

Can you review my changes for adding the workflow support for account roles? In order to fully support account roles in the workflow, it requires modification to the workflow definition schema. We won't be able to create the new schema file until 7.3 is released so we'll need to hold off on sending the changes. Yet, I'd like to get your review first to make sure the changes are good so we can send it as soon as 7.3 is released. Thanks\!

### CODE REVIEW

File: definitions/liferay-workflow-definition_7_3_0.xsd [hunk 1]
```diff
@@ -451,6 +451,7 @@
 	</xs:simpleType>
 	<xs:simpleType name="role-type">
 		<xs:restriction base="xs:string">
+			<xs:enumeration value="account" />
```
Comment (pei-jung) [inline definitions/liferay-workflow-definition_7_3_0.xsd:454]: This is temporary change. We'll need to create a new schema `liferay-workflow-definition_7_4_0.xsd` and apply the change.

File: modules/apps/roles/roles-admin-api/src/main/java/com/liferay/roles/admin/role/type/contributor/RoleTypeContributor.java [hunk 2]
```diff
@@ -109,6 +109,10 @@ public default String getClassName() {
 	 */
 	public int getType();
 
+	public default String getTypeLabel() {
+		return getName();
+	}
```
Comment (drewbrokke) [inline modules/apps/roles/roles-admin-api/src/main/java/com/liferay/roles/admin/role/type/contributor/RoleTypeContributor.java:114]: Would it be reasonable to instead use `RoleConstants.getTypeLabel(getType())` as the default? If we did that, we wouldn't have to update any of implementations.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Hey Pei, overall this looks good to me. I only left one comment of something I think should change.

Are there any other places we can add test coverage for this feature? I'm not familiar with it so there may not be, but I figured I'd at least ask.

Overall, looks good!

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
see https://github.com/pei-jung/liferay-portal/pull/711.

**Pull request closed without merging**